catalog/pipeline/fbc-release/0.11/fbc-release.yaml

```diff
@@ -154,30 +149,45 @@ spec:
               value: $(params.requester)
             - name: config_map_name
               value: $(params.signingConfigMapName)
+      when:
+        - input: "$(tasks.add-fbc-contribution-to-index-image.results.extraData)"
+          operator: in
+          values: ["fbc_opt_in=true"]
       runAfter:
```

Comment:
This is no longer needed, since the task now relies on that other task's result, so it will wait for it automatically. While there is no harm in leaving it there, I would probably remove it.

Reply:
indeed. block removed.